addons/content-index/src/main/java/org/commonjava/indy/content/index/IndexingContentManagerDecorator.java

```diff
@@ -599,6 +580,26 @@ public Transfer getTransfer( final StoreKey storeKey, final String path, final T
         return transfer;
     }
 
+    private void nfcForGroup( final ArtifactStore store, final Transfer transfer, final ConcreteResource resource )
```

Comment:
@jdcasey I'm afraid you're right about the case. I've created the ftest and it failed in last step, as the path can not even fetched via A because the NFC not cleared in A. So I'm still thinking your suggestion in terms of the fix, and will push that later.